src/xb-builder.c

```diff
@@ -803,17 +802,20 @@ xb_builder_compile(XbBuilder *self,
 		XbBuilderSource *source = g_ptr_array_index(priv->sources, i);
 		const gchar *prefix = xb_builder_source_get_prefix(source);
 		g_autofree gchar *source_guid = xb_builder_source_get_guid(source);
-		g_autoptr(XbBuilderNode) root = NULL;
+		XbBuilderNode *root = NULL;
```

Comment:
This kind of API break is a huge pain for people using the library. I really think we should avoid changing how people are using the public libxmlb API.